status.html.erb

```diff
@@ -1,10 +1,24 @@
 <html>
 <head>
-<link rel="stylesheet" href="style.css" />
+  <link href="https://solidus.io/stylesheets/solidus-90f13851.css" rel="stylesheet" type="text/css">
```

Comment:
Yes. Could we add a deployment step that adds a symlink from the current style file to a file without the cache busting hash? Then we could point to that file here.